frontend/src/components/panels/actionitems/WeeklyActionItemsPanel.tsx

```diff
@@ -0,0 +1,51 @@
+import { Paper, Typography } from '@mui/material';
+import ActionItemsTable, { ActionItemsColumn } from './ActionItemsTable';
+import { IActionItem } from 'shared-types';
+import ActionItemCheckbox from './ActionItemCheckbox';
+import './WeeklyActionItemsPanel.css';
+
+interface ActionItemsPanelProps {
+  data: IActionItem[];
+  height: number;
+  itemsID: string;
+  updateActionItemStatus: (item: IActionItem, itemsID: string) => void;
```

Comment:
replace this function with onComplete. The onComplete function will get the item and you will send the onComplete function from home page together with the data. this way you don't need the itemsID props